src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
+    private static final Lottery lottery = new Lottery();
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
```

Comment:
Please remove the redundant empty line to keep your code clean and consistent with the coding style (checklist item: Please don't add redundant empty lines to your code).